tests/providers/google/cloud/hooks/test_gcs.py

```diff
@@ -672,7 +672,7 @@ def test_download_to_file(self, mock_service):
         )
 
         self.assertEqual(response, test_file)
-        download_filename_method.assert_called_once_with(test_file)
+        download_filename_method.assert_called_once_with(test_file, timeout=60)
```

Comment:
How about swapping the `timeout=60` with a variable as well? So that we don't need to hard code them in all test cases

Reply:
I think it's good to have them hard-coded :).

the tests should be DAMP rather than DRY https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests